src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,17 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.1
+
+This is a bugfix release:
+
+* The default image type for a dedicated gateway node is changed to a smaller one for OpenStack Cloud prepare.
```

Comment:
Can we specify what is the actual type? "smaller one" is not very descriptive IMO.

Reply:
Initially we used to use the following image `PnTAE.CPU_16_Memory_32768_Disk_80` and it was changed to `PnTAE.CPU_4_Memory_8192_Disk_50` - i.e., 4 VCPUs, 8GB RAM, 50GB disk